api.md

```diff
@@ -170,6 +171,114 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
 SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_exists => true);
 ```
 
+---
+## add_data_node() :community_function: [](add_data_node)
+
+WHen executed on the access node, this will add a new data node to the
```

Comment:
Am I correct that adding the first data node makes the current node to become an access node? Can it be clarified what it means to call it on the node the first time?

Reply:
This might be good place to give idea to users what an access node is, or at a function description where it is created.

Reply:
Double capital in `WHen`

Reply:
@k-rus There is a need for a separate description of what access nodes and data nodes are, but I think this is beyond the scope of this (reference) documentation.

IMHO, nodes as such as not dedicated data or access nodes, so it does not "become" an access node. For a distributed hypertable, there is one access node where the table is accessed as a whole, and data nodes where the pieces of the table is, so a node can be both an access node and a data node depending on how the hypertable was created.

Reply:
I have the same understanding. Thus the phrase

> When executed on the access node

is confusing for me. Can it be better something like:

> Add a data node into the current database to be used with distributed hypertables through this access node.

Reply:
Reformulated.